lib/algos/list/delimiterMaster.js

```diff
@@ -62,7 +63,8 @@ class DelimiterMaster extends Delimiter {
         let key = obj.key;
         const value = obj.value;
 
-        if (key === this.prefix) {
+        if (key === this.prefix || key[key.length - 1] === this.delimiter) {
```

Comment:
Why do we have `key[key.length - 1] === this.delimiter` as an extra condition? It does not look right to me to say that we've seen the prefix key when we actually have seen a key that ends with a delimiter.